patches/@types+invity-api+1.0.6.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/node_modules/@types/invity-api/index.d.ts b/node_modules/@types/invity-api/index.d.ts
```

Comment:
Uff more patches, why not publish new version of your package with fixed types?

Reply:
@martinboehm, could you please answer this one?

Reply:
@Nodonisko this is only a small fix in the types. We will release a new version of the types with the next bigger feature in about one month, the fixup is only very temporary.